INSTALL.md

```diff
@@ -294,13 +303,13 @@ with the real build process. These preprations can be done as follows:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 $ git clone git://github.com/Secretchronicles/TSC.git
 $ cd TSC
-$ git submodule init
-$ git submodule update
+$ git submodule init; git submodule update
```

Comment:
Please change `git submodule init` and `git submodule update` to be at separate lines like they were previously, it's more clear.